src/lib/web-component/filter-option/filter-option-element.js

```diff
@@ -95,6 +82,29 @@ class FilterOption extends HTMLElement {
     this.toggleBtn.classList.toggle('expanded');
   }
 
+  handleCheckbox(checkboxLabel) {
```

Comment:
I suggest to use `onCheckboxChange`. And also pass the value true or false, depending on whether the checkbox is checked, so you can use it instead of `.has(...)` in the if below.